src/Compilers/Core/Portable/InternalUtilities/StringTable.cs

```diff
@@ -63,7 +65,11 @@ private struct Entry
         // slightly slower than local cache
         // we read this cache when having a miss in local cache
         // writes to local cache will update shared cache as well.
+#if COMPILERCORE
```

Comment:
note: i could make this change for the compiler as well, but i wasn't sure if tehre would be risk aversity around this core data structure.